lib/ex_unit/lib/ex_unit.ex

```diff
@@ -176,6 +178,8 @@ defmodule ExUnit do
     * `:assert_receive_timeout` - the timeout to be used on `assert_receive`
       calls. Defaults to 100ms.
 
+    * `:autorun` - if ExUnit should run by default on exit; defaults to `true`
```

Comment:
This setting was just moved so I didn't actually read it. It think both works here but "whether" is a bit more formal.